components/camera/camera.go

```diff
@@ -147,6 +154,22 @@ func NewFromReader(
 	}, nil
 }
 
+// NewPinholeCameraModel creates a transform.PinholeCameraModel from
+// a *transform.PinholeCameraIntrinsics and a *transform.BrownConrady.
+// If *transform.BrownConrady is `nil`, transform.PinholeCameraModel.Distortion
+// is not set & remains nil, to prevent https://go.dev/doc/faq#nil_error.
+func NewPinholeCameraModel(pinholeCameraIntrinsics *transform.PinholeCameraIntrinsics,
+	distortion *transform.BrownConrady,
```

Comment:
What about when we add support for other distortion models that are not Brown Conrady? Can you maybe rename the helper function to specify its with a BrownConrady type distortion?

Ie NewPinholeModelWithBrownConradyDistortion

Reply:
Done